### DISCUSSION TITLE
[4.0] Enable rearranging of module item types in admin control panel and make headings clickable

### DISCUSSION THREAD

**Discussion by Scrabble96** [joomla/joomla-cms#31630] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
Not a problem but two requests:

Select content > modules > new
1. The module types are listed in alphabetical order. 
2. They are very large so there's a lot of scrolling to find your chosen type if not at the top
3. The titles are not clickable

### Describe the solution you'd like
1. Have the option to rearrange the module types so that the ones used most often are at the top
2. Remove some of the space between the title and the description. There's a double set of adjacent padding.
3. Make the titles clickable

**Comment by chmst** [comment]
Thank you. 
About scrolling - the searchfield on top makes it easier to find the desired module. But your suggestions are welcome. I make this a feature request. <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31630">issues.joomla.org/tracker/joomla-cms/31630</a>.</sub>

**Comment by Scrabble96** [comment]
> 
> 
But your suggestions are welcome. I make this a feature request.
> This comment was created with the [J!Tracker Application](https://github.com/joomla/jissues) at [issues.joomla.org/tracker/joomla-cms/31630](https://issues.joomla.org/tracker/joomla-cms/31630).

Thank you.

**Comment by adj9** [comment]
You can think of making another kind of FrontEnd with are a list with the name of the module, which is clickable, and the description.

By placing a flag in the global imports of J!, you choose the type of schema to use.

Maybe trails off the standards used.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/31630">issues.joomla.org/tracker/joomla-cms/31630</a>.</sub>
